### PR TITLE
OPSEXP-1776: refactor dbp verify helm action to allow for keeping logs

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1776

### CODE REVIEW

File: .github/actions/dbp-charts/verify-helm/helm_delete.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash +e
+#
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-helm/helm_delete.sh:2]:
```suggestion
```

File: .github/actions/kubectl-keep-nslogs/action.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      shell: bash
+      run: |
+        ARTIFACTS_DIR=$(mktemp -d)
+        echo "ARTIFACTS_DIR=${ARTIFACTS_DIR}" >> $GITHUB_OUTPUT
+    - name: Fetch logs
+      shell: bash
+      working-directory: ${{ steps.artifactdir.outputs.ARTIFACTS_DIR }}
+      run: |
+        echo "checking pwd (should be ${{ steps.artifactdir.outputs.ARTIFACTS_DIR }})"
+        pwd
```
Comment (gionn) [inline .github/actions/kubectl-keep-nslogs/action.yml:29]:
```suggestion
```

trust no one xD

File: .github/actions/kubectl-keep-nslogs/action.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          for i in $(kubectl get "$type" -o name); do \
+            kubectl logs "$i" --all-containers=true | \
+            gzip > "${i}.log.gz"
+          done
+        done
+    - name: upload kubernetes logs as artifact
+      uses: actions/upload-artifact@v3.1.1
+      with:
+        name: >-
+          ${{ inputs.namespace }}_logs.${{ github.run_number }}.${{ github.run_attempt }}
```
Comment (gionn) [inline .github/actions/kubectl-keep-nslogs/action.yml:42]: do you mean that artifacts are shared between reruns? 🤔

Comment (alxgomz) [reply .github/actions/kubectl-keep-nslogs/action.yml:42]: I though of using `github.run_id` but found that \run_number` would be more "readable", but now that you say it, I feel like I'm replcating the behavior that was upetting in travis but for kubernetes logs :)

Comment (gionn) [reply .github/actions/kubectl-keep-nslogs/action.yml:40]: ok I think that each run has its own artifacts, however it's harmless having this so ok

File: .github/actions/dbp-charts/verify-helm/action.yml [hunk 4]
```diff
@@ -89,14 +89,35 @@ runs:
     - name: Get commit msg
       uses: >-
         Alfresco/alfresco-build-tools/.github/actions/get-commit-message@v1.20.3
+    - name: Get a namespace to deploy on
+      id: k8sns
+      uses: >-
+        Alfresco/alfresco-build-tools/.github/actions/dbp-charts/kubernetes-valid-ns@OPSEXP-1776
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-helm/action.yml:95]: remember that you probably want to immediately do a new release changing those references in the release PR after merging this PR

**Review by gionn — APPROVED** [review]

**Pull request merged**